module/combat-degenesis.js

```diff
@@ -51,41 +57,38 @@ export class DegenesisCombat extends Combat {
         if (!actor) return 0;
         this.resetInitiativeState(actor);
         const skill = DEGENESIS.fightRolls.initiative;
+        const spentEgo = actor.data.data.state.spentEgo.value;
         const rollData = {
             skill: actor.data.data.skills[skill],
-            actionNumber: actor.data.data.attributes[actor.data.data.skills[skill].attribute].value + actor.data.data.skills[skill].value
+            actionNumber: actor.data.data.attributes[actor.data.data.skills[skill].attribute].value + actor.data.data.skills[skill].value + spentEgo
         };
-        const rollResults = DegenesisDice.rollWithout3dDice(rollData);
+        const {rollResults, cardData} = actor.rollFightRollSync("initiative");
```

Comment:
Since I assumed you removed this due to spent ego triggers not counting, I added it back since we determined that is not the case.